src/main/resources/com/hcl/appscan/jenkins/plugin/scanners/DynamicAnalyzer/config_en.properties

```diff
@@ -4,10 +4,15 @@
 label.starting.url=Starting URL
 label.additional.options=Additional Options
 label.scan.type=Scan Type
-label.scan.optimization=Test Optimization 
-label.login.user=Login User
-label.login.password=Login Password
+label.scan.optimization=Test Optimization
 label.extra.field=Third Credential
 label.presence.id=Presence
 label.scan.file=Scan File
+label.login.management= Which Login method you want to use?
```

Comment:
A minor change to this message: Which login method **do** you want to use?

Reply:
Okay, will make the changes as suggested.